bHY6yh-d1e792/cod-acbnn_bHY6yh-d1e792.xml

```diff
@@ -0,0 +1,197 @@
+                hominis dirigentur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e131">
+               // <ref xml:id="bHY6yh-d1e792-Rd1e128">Idem lxxxiii</ref> <quote xml:id="bHY6yh-d1e792-Qd1e133">
+              <lb ed="#A"/>Gratiam et gloriam dabit dominus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e142">
+               <ref xml:id="bHY6yh-d1e792-Rd1e142">Idem xvii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e144">
+               <lb ed="#A"/>Lapidem quem reprobauerunt edificantes hic 
+               <lb ed="#A"/>factus est in caput anguli. A domino factum est 
+               <lb ed="#A"/>istud et est mirabile in oculis nostris. Hec
+               <lb ed="#A"/> est dies quam fecit dominus exultemus et letemur
+               <lb ed="#A"/> in ea.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e156">    
+               // <ref xml:id="bHY6yh-d1e792-Rd1e160">Idem xlviio.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e160">Nisi dominus custodierit
+               <lb ed="#A"/>ciuitatem / frustra vigilant qui custoderit 
+               <lb ed="#A"/>eam.
+                  <!--Idem lxvi in this edition--> Dominus dat escam vniuerse
+                  <lb ed="#A"/>carni.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e168">
+               omitted
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e176">
+               <lb ed="#A"/><ref xml:id="bHY6yh-d1e792-Rd1e185">Salomon in prouerbiis. c. iii.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e180">habe fiduciam 
+               <lb ed="#A"/>in domino ex toto corde / et ne initaris pru
+               <lb ed="#A"/>dentie tue. In omnibus viis cogita illum et 
+               <lb ed="#A"/>ipse diriget gressus tuos.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e190">
+               <ref xml:id="bHY6yh-d1e792-Rd1e199">Idem. c. xvi.</ref>  <quote xml:id="bHY6yh-d1e792-Qd1e192">
+               <lb ed="#A"/>Cor hominis disponet viam suam / sed domini est
+               <lb ed="#A"/>dirigere gressus eius.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e200">
+               <quote xml:id="bHY6yh-d1e792-Qd1e202"> Ibidem a domino 
+               <lb ed="#A"/>diriguntur gressus viri. Quis autem hominum 
+               <lb ed="#A"/>intelligere potest viam suam.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e211">
+              Omitted
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e219">    
+                <ref xml:id="bHY6yh-d1e792-Rd1e234">Paralipomenon. l. 2<!--o.?--> c. xxv.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e223">dei est 
+                <lb ed="#A"/>adiuuare et vertere in fugam.</quote>
+            </p>
+            <p xml:id="bHY6yh-d1e792-d1e229">         
+                <lb ed="#A"/><ref xml:id="bHY6yh-d1e792-Rd1e246">Ioannes in euangelio. c. vi.</ref> <quote xml:id="bHY6yh-d1e792-Qd1e233">Ait yesus: Nemo 
```

Comment:
<ref>Ioannes in evangelio c. vi</ref> Ait yesus <quote>nemo potest venire...</quote>